lib/vanagon/component/source/http.rb

```diff
@@ -105,7 +105,7 @@ def extract(tar)
             end
           else
             # Extension does not appear to be an archive
-            return nil
+            return ':'
```

Comment:
Works for me. We'll hammer it out somewhere else and fix it in `MASTER` when we decide.